run_project.md

```diff
@@ -0,0 +1,44 @@
+# How to run the project after installing everything
```

Comment:
The header says, "How to run the project after install everything," but this file seems only to contain miscellaneous tips, and I find some inconsistency here. Maybe we should rename the title of the file to imply that this document will have some tips for begineers

Reply:
Thank you!